_infra/helm/secure-message-v2/values.yaml

```diff
@@ -54,5 +54,9 @@ dns:
 crons:
   markThreadForDeletion:
     name: sm-v2-mark-thread-for-deletion-scheduled-job
-    schedule: "0 1 * * *"
+    schedule: "0 2 * * *"
```

Comment:
Tweaked, so it marks it one day then 23 hours later it deletes it

Reply:
Good idea. Do we want the cron jobs running everyday or just monday to friday?

Reply:
I don't think it needs to be explicit at the moment, I have made it configurable so when we do decide to go live we can adjust it to what is appropriate in each environment

Reply:
Agreed, keep it consistent with the other deletion activities that have a 23 hour grace period. Although we may need to change it briefly to facilitate initial business testing depending on their approach.